examples/image_classifier/resnet_152_batch/resnet152_handler.py

```diff
@@ -1,93 +1,25 @@
-import io
-import logging
-import numpy as np
-import os
 import torch
-from PIL import Image
-from torch.autograd import Variable
-from torchvision import transforms
 
-logger = logging.getLogger(__name__)
+from ts.torch_handler.image_classifier import ImageClassifier
 
 
-class BatchImageClassifier(object):
+class BatchImageClassifier(ImageClassifier):
     """
```

Comment:
This can be part of another change, but it would be good to refactor BaseHandler to handle batching in `preprocess` & `inference`. I would suggest we make the assumption that all models used (speaking of the torch.nn.Module) support batch inputs, and offer an opt-out option to those that can;t.

Reply:
@htappen Yes, as commented on your proposal #434, it can be taken up.